### ISSUE TITLE
During install Error : Vesta.servicejob for vesta.service failed 

### ISSUE THREAD

**Issue by mehargags** [serghey-rodin/vesta#1712]
OS : Debian 9.5
VestaCP Release : 22
Apache+Nginx
Fresh install on 4 different servers shows this error during early installation process. Looks like the services/configs are still to be done and something tries to restart the service which throws the "fail" error.
Attached screenshot.
![vesta-error](https://user-images.githubusercontent.com/2650395/45842722-2833be80-bd3b-11e8-98c1-b637e3507149.jpg)

**Comment by dpeca** [comment]
Important information to say - vesta is normally running when installation is done.
This error is just in the middle of installation.
No other impacts.

**Comment by Skamasle** [comment]
Yes no problem with that, but will be interesting know what happen, maybe add a log in installer and test, or just add a line to show a status before that and know the reason, just for debuging
